dts/bindings/dma/brcm,iproc-pax-dma-v2.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Broadcom
+# SPDX-License-Identifier: Apache-2.0
+
+description: Broadcom iProc PAX(PCIE<->AXI) DMA controller version 2
+
+include: dma-controller.yaml
+
+compatible: brcm,iproc-pax-dma-v2
+
+properties:
+    reg:
+      description: |
+        Register space for the memory mapped PAX DMA controller registers,
+        It includes data mover engine(dme), ring manager, ring manager common
+        registers.
+      required: true
+    label:
+      required: true
+    bd-memory:
```

Comment:
This isn't memory that is part of this peripheral, right? Instead, it describes preallocated memory on the system?

Reply:
This memory is not part of peripheral, it's pre allocated memory on the system.